src/screens/AbcCams/AbcCams/styles.scss

```diff
@@ -0,0 +1,19 @@
+.main {
+  text-align: center;
+  background-color: #F8F8F8;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  min-height: 70vh;
+}
+
+.footer {
+  display: flex;
+  background-color: #282828;
```

Comment:
Colors should be imported from `colors.scss`

Reply:
Done